src/orquestra/sdk/_base/abc.py

```diff
@@ -194,6 +194,12 @@ def get_workflow_logs(
         """
         raise NotImplementedError()
 
+    def get_workflow_project(self, wf_run_id: WorkflowRunId):
+        """
+        Returns project and workspace IDs of given workflow
+        """
+        raise WorkspacesNotSupportedError()
```

Comment:
**suggestion**: please don't rely on inheriting the method implementation, this ABC was meant only for specifying the interface to adhere to.
```suggestion
        raise NotImplementedError()
```